frontend/src/components/prompt/PopoverPrompt.vue

```diff
@@ -96,16 +97,15 @@ export default {
       return positions
     },
     contentClass() {
-      return (
-        'ec-popover-prompt ' +
-        (this.x === 'left'
+      return `ec-popover-prompt ${
+        this.x === 'left'
           ? this.y === 'bottom'
             ? 'ec-popover-prompt--topright'
             : 'ec-popover-prompt--bottomright'
           : this.y === 'bottom'
           ? 'ec-popover-prompt--topleft'
-          : 'ec-popover-prompt--bottomleft')
-      )
+          : 'ec-popover-prompt--bottomleft'
+      }`
```

Comment:
can we have this with normal if statements instead of 3 ternary statements in a string concatenation?
Maybe not in this PR

Reply:
Fixed in #3789